attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/AttributeServiceImplTest.java

```diff
@@ -504,6 +507,10 @@ private Document createMockDocument(
         .append(kind.name())
         .append("\",\"tenant_id\":\"")
         .append(tenantId)
+        .append("\",\"description\":\"")
+        .append(scope.name())
```

Comment:
We are copying the `scope.name()` as the description. 
Can we keep the existing tests as they are and add an additional test for the description? Since the `description` string is optional, the existing tests should work fine, right?

Reply:
You are right, I've undone any changes I made to this file. Do we need separate tests for description apart from the ones added in AttributeMetadataModelTest?